doc/releases.md

```diff
@@ -116,7 +121,9 @@ This macro is used to signal an ABI version for native addons. It currently has
 
 The general rule is to bump this version when there are _breaking ABI_ changes and also if there are non-trivial API changes. The rules are not yet strictly defined, so if in doubt, please confer with someone that will have a more informed perspective, such as a member of the NAN team.
 
-### 6. Create Release Commit
+**Note** that it is current TSC policy is to bump major version when ABI changes so if you see a need to bump `NODE_MODULE_VERSION` then you may also need to bump the major version. Consult the TSC in this case.
```

Comment:
How about something like "don't bump the `NODE_MODULE_VERSION` until a new major release"?